content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -5,53 +5,139 @@ weight = 50
 +++
 
 This page describes authentication for Kubeflow Pipelines to GCP.
+Available options listed below have different tradeoffs. You should choose the one that fits your use-case.
+
+* [Compute engine default service account](#compute-engine-default-service-account) is easy to set up, but overgrants permission. Therefore, it is not suitable for a shared GCP project.
+* [Workload Identity](#workload-identity) takes more efforts to set up, but allows fine-grained permission control. It is recommended for production use-cases.
```

Comment:
"It is recommended for production use-cases."
Since it's not so ready in KFP, how about we remove this sentence for now?

Reply:
I believe it's ready for standalone and Full Kubeflow. Any specific problem you think it's not ready?

Reply:
I mean CloudSQL/Minio-GCS still require ADC. It's OK we handle it in separate PR (not in this repo)

Reply:
will handle cloudsql and minio documentaiton separately